### PR TITLE
Create a device label fixes #32

### LINKED ISSUE #32: [Hardware-Wallet]: create a device label
Device Label is a "name" that can appear on the screen and be used to identify a device.

Supporting this feature now is not complicated but requires a bunch of extra specifications like:
* shall we create a specific message to label the device ?
* how do we display it on the screen ?

### PR DESCRIPTION
**Pull request by stdevAlDen**
closes #32 

Changes:
- Use a label to identify the device on the screen (if not label is defined returns the same value of `device_ID`).

Does this change need to mentioned in CHANGELOG.md?
yes

Requires testing
yes

### CODE REVIEW

File: tiny-firmware/emulator/flash.c [hunk 1]
```diff
@@ -20,6 +20,7 @@
 #include <string.h>
 #include <assert.h>
 #include <stdbool.h>
+#include <unistd.h>
```
Comment (olemis) [inline tiny-firmware/emulator/flash.c]: this is the only change here . what is it for ?

Comment (stdevAlDen) [reply tiny-firmware/emulator/flash.c]: for `ssize_t` data type

File: tiny-firmware/firmware/layout2.c [hunk 2]
```diff
@@ -114,12 +113,9 @@ void layoutHome(void)
...
 		oledDrawBitmap(0, 0, &b);
 	} else {
-		if (label && strlen(label) > 0) {
-			oledDrawBitmap(44, 4, &bmp_logo48);
-			oledDrawStringCenter(OLED_HEIGHT - 8, label, FONT_STANDARD);
-		} else {
-			oledDrawBitmap(40, 0, &bmp_logo64);
-		}
+		oledDrawBitmap(44, 4, &bmp_logo48);
+		oledDrawStringCenter(OLED_HEIGHT - 8, _("Go to www.skycoin.net"),
```
Comment (olemis) [inline tiny-firmware/firmware/layout2.c]: Why the change ? Shouldn't this be `oledDrawStringCenter(OLED_HEIGHT - 8, label, FONT_STANDARD);` where `label = storage_isInitialized() ? storage_getLabel() : _("Go to trezor.io/start");¸` ?

File: tiny-firmware/firmware/layout2.c [hunk 3]
```diff
@@ -128,7 +124,22 @@ void layoutHome(void)
 		if (storage_needsBackup()) {
 			oledBox(0, 0, 127, 8, false);
 			oledDrawStringCenter(0, "NEEDS BACKUP!", FONT_STANDARD);
-		}
+		} else {
+				// NOTE(denisacostaq@gmail.com): The scree is not long enough, so cut the
+				// label string.
+				char label[MAX(sizeof (storageUpdate.label), sizeof (storage_uuid_str))];
+				_Static_assert(sizeof(storageUpdate.label) > 23
```
Comment (olemis) [inline tiny-firmware/firmware/layout2.c]: not sure about this ...

**Comment by olemis** [comment]
@stdevAlDen conflicts
